client/components/multiple-choice-question/index.jsx

```diff
@@ -25,26 +25,28 @@ const MultipleChoiceQuestion = ( {
 	question,
 	selectedAnswerId,
 	selectedAnswerText,
+	shouldShuffleAnswers,
 } ) => {
 	const [ selectedAnswer, setSelectedAnswer ] = useState( selectedAnswerId );
-	const shuffledAnswers = shuffleAnswers( answers );
-
+	const shuffledAnswers = shouldShuffleAnswers ? shuffleAnswers( answers ) : answers;
 	return (
 		<FormFieldset className="multiple-choice-question">
 			<FormLegend>{ question }</FormLegend>
-			{ shuffledAnswers.map( ( answer ) => (
-				<MultipleChoiceAnswer
-					key={ answer.id }
-					answer={ answer }
-					disabled={ disabled }
-					isSelected={ selectedAnswer === answer.id }
-					onAnswerChange={ ( id, textResponse ) => {
-						onAnswerChange( id, textResponse );
-						setSelectedAnswer( id );
-					} }
-					selectedAnswerText={ selectedAnswer === answer.id ? selectedAnswerText : '' }
-				/>
-			) ) }
+			<div className="multiple-choice-answers-wrapper">
```

Comment:
As per [BEM](http://getbem.com/introduction/), it should be

```suggestion
			<div className="multiple-choice-question__answers">
```

or

```suggestion
			<div className="multiple-choice-question__answers-wrapper">
```